### PR TITLE
Add support for host checking

### LINKED ISSUE #247: Add support for host checking
**Is your feature request related to a problem? Please describe.**
I would like to have strict host checking enabled. Currently it is hard coded to disabled [here](https://github.com/butlerx/wetty/blob/6654e1a7d71236d4b56425359c8e9fa080bcafd8/src/server/command/ssh.ts#L19) and [here](https://github.com/butlerx/wetty/blob/6654e1a7d71236d4b56425359c8e9fa080bcafd8/src/server/command/ssh.ts#L21).

**Describe the solution you'd like**
I would like an additional flag along the lines of `--knownhosts`. It can default to `/dev/null` to keep the current behavior, but if it is provided and set to something else, `StrictHostKeyChecking=yes` should be set as well.

I think that is the simplest way, but two flags would also work. `--knownhosts` which would default to `/dev/null` and `--stricthostchecking` which would default to `no`.

**Describe alternatives you've considered**
Currently, there are no alternatives to having strict host checking disabled.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by kmlucy**
Fixes #247

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,6 +73,12 @@ if (require.main === module) {
           type: 'boolean',
           default: process.env.FORCESSH || false
         },
+        knownhosts: {
+          demand: false,
+          description: 'path to known hosts file',
+          type: 'string',
+          default: process.env.KNOWNHOSTS || '/dev/null',
```
Comment (butlerx) [inline index.js:80]: would it be better to use `~/.ssh/known_hosts`

File: src/server/command/ssh.ts [hunk 2]
```diff
@@ -3,10 +3,16 @@ import parseCommand from './parse';
 import logger from '../utils/logger';
 
 export default function sshOptions(
-  { pass, path, command, host, port, auth }: { [s: string]: string },
+  { pass, path, command, host, port, auth, knownhosts }: { [s: string]: string },
   key?: string
 ): string[] {
+  let hostChecking;
   const cmd = parseCommand(command, path);
+  if (knownhosts !== '/dev/null') {
```
Comment (butlerx) [inline src/server/command/ssh.ts]: Can you switch this to a const using a ternary

Comment (kmlucy) [reply src/server/command/ssh.ts]: Fixed with https://github.com/butlerx/wetty/pull/250/commits/079f5ac3b3bfee334011c2b45c50c06b6ac227ac

File: index.js (hunk 1 above)
Comment (kmlucy) [reply index.js:80]: I was trying to leave the existing functionality intact. I think leaving it as `/dev/null` makes more sense with the way I have strict host checking set. This way, if people want to use `~/.ssh/known_hosts`, they can just provide that option and will have strict host checking enabled.

If it defaults to `~/.ssh/known_hosts`, there will be no way to have strict host checking enabled with that file without adding another flag.

**Review by butlerx — APPROVED** [review]

**Comment by kmlucy** [comment]
@butlerx Can you create a new tag/release? That way I can update my PR for [svenihoney/docker-wetty-alpine](https://github.com/svenihoney/docker-wetty-alpine/) to use the new flag.

**Pull request merged**